arkouda/random/_generator.py

```diff
@@ -209,6 +209,55 @@ def integers(self, low, high=None, size=None, dtype=akint64, endpoint=False):
         self._state += size
         return create_pdarray(rep_msg)
 
+    def normal(self, loc=0.0, scale=1.0, size=None):
+        r"""
+        Draw samples from a normal (Gaussian) distribution
```

Comment:
The doc string for this one isn't rendering for me with `make doc`.

Reply:
oh really?? that's odd. I might swing by to see what it's showing you

Reply:
yah I found the problem... just had a random `latex` keyword in the usage section `.. autofunction:: latex arkouda.random.Generator.normal`... should be fixed now!